CodingTracker.obitom67/DBHandling.cs

```diff
@@ -0,0 +1,141 @@
+﻿global using System.Configuration;
+using HabitTracker_obitom67;
+using System;
+using System.Globalization;
+using Microsoft.Data.Sqlite;
+using Dapper;
+using Spectre.Console;
+
+
+namespace CodingTracker.obitom67
+{
+    internal static class DBHandling
+    {
+        
+
+        public static void TableHandling()
+        {
+            string connectionString = ConfigurationManager.AppSettings.Get("key1");
+            using (var connection = new SqliteConnection(connectionString))
+            {
+                connection.Open();
+
+                var tableCMD = connection.CreateCommand();
+                tableCMD.CommandText =
+                    @"CREATE TABLE IF NOT EXISTS CodingSessions (
+                        Id INTEGER PRIMARY KEY AUTOINCREMENT,
+                        StartTime TEXT,
+                        EndTime TEXT,
+                        Duration TEXT
+                        )";
+
+                tableCMD.ExecuteNonQuery();
+                connection.Close();
```

Comment:
He i think you are using Ado.Net here instead of Dapper. 
Can you switch to Dapper alternative for create table query please.